mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/index/MongoDBIndexTest.java

```diff
@@ -39,8 +42,9 @@ public void testSuccessfulIndexCreation() throws Exception {
 				indexMap.get( "name_idx" ).toJson() );
 		assertJsonEquals( "{ 'v' : 2 , 'unique' : true , 'key' : { 'author' : 1 , 'name' : 1} , 'name' : 'author_name_idx' , 'ns' : 'ogm_test_database.T_POEM' , 'sparse' : true}",
 				indexMap.get( "author_name_idx" ).toJson() );
-		assertJsonEquals( "{ 'v' : 2 , 'key' : { 'name' : 1 , 'author' : 1} , 'name' : 'IDXjo3qu8pkq9vsofgrq58pacxfq' , 'ns' : 'ogm_test_database.T_POEM' }",
-				indexMap.get( "IDXjo3qu8pkq9vsofgrq58pacxfq" ).toJson() );
+
+		// Index IDXjo3qu8pkq9vsofgrq58pacxfq will be never defined
```

Comment:
This comment is not very useful if don't explain why.
In this scenario, I prefer to use the `.as("")` so that you can explained what made the assertion fails.

Reply:
Keep in mind that you are changing the code. so you know what that index is about. But when I  look at the  source code only I only see `IDXjo3qu8pkq9vsofgrq58pacxfq`  and I don't know what index is supposed to define and we we don't expect to exists. Also, what if it failes because the name is changed?

I know that you didn't write this test originally, but I think it needs to be updated. For example, one assertion state that the size is 5 but it only checks 3 of the indexes. Do you think it is possible to improve it a bit? I genuinely asking because I don't know what the other values in the map are